eZ/Bundle/EzPublishCoreBundle/Resources/views/default/embed.html.twig

```diff
@@ -0,0 +1,3 @@
+{% for field in content.FieldsByLanguage %}
+    {{ ez_render_field(content, field.fieldDefIdentifier) }}
+{% endfor %}
```

Comment:
I don't think we want to iterate all fields on embed *(and line, which should imho be more or less exact same with different class assign to it so it can be styles differently)*.

Minimum is to do like in legacy:
https://github.com/ezsystems/ezdemo/blob/master/packages/ezdemo_extension/ezextension/ezdemo/design/ezdemo/templates/node/view/embed.tpl

What we can optionally also do is output intro/summary field if defined as a convention.